lldb-eval/api.h

```diff
@@ -45,6 +45,9 @@ namespace lldb_eval {
 struct ContextVariable {
   const char* name;
   lldb::SBValue value;
+
+  ContextVariable(const char* name, lldb::SBValue value)
+      : name(name), value(value) {}
```

Comment:
Oh, I didn't realize `emplace_back` requires a user-defined ctor. Exported inline functions are evil (ctor in this case) -- https://devblogs.microsoft.com/oldnewthing/20140109-00/?p=2123, let's try to avoid them. I'd just revert is to `push_back`, doesn't make much difference. Sorry for confusion :)

Reply:
Reverted. I learned something new, too :)